src/api/app/models/obs_factory/openqa_api.rb

```diff
@@ -59,5 +39,24 @@ def get(url, params = {}, options = {})
       end
       ActiveSupport::JSON.decode(resp.body)
     end
+
+    private
+
+    # A get that follows redirects - openqa redirects to https
+    def _get(uri, counter_redirects)
+      req_path = uri.path
+      req_path << "?" + uri.query if uri.query.present?
+      req = Net::HTTP::Get.new(req_path)
+      resp = Net::HTTP.start(uri.host, use_ssl: uri.scheme == "https") { |http| http.request(req) }
+      # Prevent endless loop in case response is always 301 or 302
+      unless counter_redirects >= 5
+        if resp.code.to_i == 302 or resp.code.to_i == 301
```

Comment:
although this is just moving code, this could be written like `[302, 301].include?(resp.code.to_i)`

Reply:
I'll do a new PR for this. Thanks!